website/content/docs/discovery/dns.mdx

```diff
@@ -359,6 +376,28 @@ Both A and SRV records are supported. SRV records provide the port that a servic
 registered on, enabling clients to avoid relying on well-known ports. SRV records are
 only served if the client specifically requests them.
 
+### Service Lookups for Consul Enterprise <EnterpriseAlert inline />
```

Comment:
I feel like this should either be:
- Above the prepared query lookups (so below the `.service` lookups), OR
- After all the lookups (including `.connect`, `.virtual`, `.ingress`)

That said, I don't think there's a perfect solution here. This isn't a strongly held belief. What are your thoughts on whether this chunk should stay here, or move to one of the other places suggested above?

Reply:
If I were to pick between the two, I would put this before prepared query lookups. After .ingress is burying it too deep.

Reply:
Ok done.